crates/bevy_window/src/window.rs

```diff
@@ -602,12 +602,26 @@ impl Window {
     }
 }
 
+/// Describes the information needed for a window creation like width, height, title, present mode, etc.
+/// This should be set up before adding the [`WindowPlugin`]. Most of these settings can also later
+/// be configured through the [`Window`] resource.
+///
+/// See `examples/window/window_settings.rs` for usage.
+///
+/// [`Window`]: crate::Window
+/// [`WindowPlugin`]: crate::WindowPlugin
 #[derive(Debug, Clone)]
 pub struct WindowDescriptor {
+    /// The requested logical width of the window's client area.
     pub width: f32,
+    /// The requested logical height of the window's client area.
     pub height: f32,
+    /// The position on the screen that the window will be placed on.
     pub position: Option<Vec2>,
     pub resize_constraints: WindowResizeConstraints,
+    /// Overrides the [`Window`]'s ratio of physical pixels to logical pixels
+    ///
+    /// [`Window`]: crate::Window
```

Comment:
I'd prefer to just embed this directly, using `[Window](crate::window)` rather than splitting it across lines.